### DISCUSSION TITLE
Parameters of the Line

### DISCUSSION THREAD

**Discussion by Cmt111** [CURENT/andes#462] [Q&A]
Dear Dr. Cui,
I am learning to use the model "ieee14_full.xlsx". In ieee14_full.xlsx, the parameters: resistance, reactance and susceptance of the Line are only r, x, b and the unit is p.u.. 
(1) Where can I find the line length?
(2) How to calculate its corresponding positive-sequence and zero-sequence resistance(Ohms/km), inductance(H/km) and capacitance(F/km)?

WINDOWS
ANDES Version 1.8.7

**Comment by jinningwang** [comment]
In ANDES, we don't have a parameter for the length in the model Line. Here are some standards providing detailed parameters that you can use to derive the resistance, etc. You can refer to the implementation of pandapower [Line](https://pandapower.readthedocs.io/en/v2.13.1/elements/line.html#input-parameters) for more details.

**Reply by cuihantao** [reply]
Thanks, Jinning.

Also, line length is not parameterized. You will need to provide the series R-X and shunt G-B directly. `Line` is the pi-model for lumped parameter lines. 

All `Line` data are positive sequence. `X = jwL` where `w = 2 * pi * 60`. Zero sequence parameters are not included or supported in ANDES.

**Reply by Cmt111** [reply]
Thank you for your response. I noticed that the unit of r, x, b of the line is p.u.. I am expecting to convert it into actual values in Ohms, H, F  It is written in pdf that "Line has parameters r, x and b as per unit values in the device bases Sn, Vn1, and Vn2". Is it calculated like this: 
Line1: R=0.01938*100=1.938Ohms  X=0.05917*69=4.08273H  B=0.0528*69=3.6432F?

**Reply by jinningwang** [reply]
Feel free to check out this discussion related to per-unit conversion, https://github.com/CURENT/andes/discussions/447

**Reply by Cmt111** [reply]
Thanks for your answer!
